cgc/kmeans.py

```diff
@@ -53,6 +53,18 @@ class Kmeans(object):
     :type kmean_max_iter: int, optional
     :param output_filename: Name of the file where to write the results.
     :type output_filename: str, optional
+
+    :Example:
+
+    >>> import numpy as np
+    >>> Z = np.array([[0, 0, 1, 1], [0, 0, 1, 1], [2, 2, 3, 3], [2, 2, 3, 3],
```

Comment:
It is not really a problem here, but for co-/tri-clustering zeros can be problematic - let's maybe avoid using them in the example:
```suggestion
    >>> Z = np.array([[4, 4, 1, 1], [4, 4, 1, 1], [2, 2, 3, 3], [2, 2, 3, 3],
```